docs/src/server-render.js

```diff
@@ -1,17 +1,25 @@
 import React from 'react';
 import { renderToString } from 'react-dom/server';
 import { StaticRouter } from 'react-router';
+import purifyCss from 'purify-css';
 import App from 'App/App';
 
 const baseHref = process.env.BASE_HREF || '/';
 
 // Static site renderer
-export default ({ path, template }) => {
+export default ({ path, template, css }) => new Promise(resolve => {
   const html = renderToString(
     <StaticRouter location={path} context={{}}>
       <App />
     </StaticRouter>
   );
 
-  return template({ html, baseHref });
-};
+  const purifyOptions = {
+    minify: true,
+    whitelist: ['*:not*'] // https://github.com/purifycss/purifycss/issues/161
+  };
```

Comment:
Might we want to log unused selectors?

Reply:
The log would be pretty noisy—I don't imagine it would help much.

Reply:
Might help in dev 🤔 But yeah, happy with either